nni/tools/nnictl/launcher.py

```diff
@@ -421,6 +421,18 @@ def launch_experiment(args, experiment_config, mode, experiment_id, config_versi
         else:
             experiment_config['searchSpace'] = ''
 
+    # Only for BOHB: The search space may contain the path to another search space file in
+    # ConfigSpace format by defining the parameter 'ConfigSpace' as a 'choice' parameter
+    # with '_value' being the path to the configuration file. If the path is relative, we
+    # need to expand it
+    if experiment_config.get('advisor') and experiment_config['advisor'].get('name') == 'BOHB'\
```

Comment:
Why do we still need this?

Reply:
To make the relative path to the ConfigSpace file an absolute one. With this code, the configSpace file can just reside in the `trialCodeDirectory` and be referenced by name. Without it, one would have to define the complete path.
The comment is deprecated, though, will change it.

Reply:
I have now moved that part to `launcher_utils.py` because this is where other paths are handled as well.